src/components/pages/CurriculumInfo/tabs/OverviewTab/OverviewTab.tsx

```diff
@@ -145,9 +145,9 @@ const OverviewTab: FC<OverviewTabProps> = (props: OverviewTabProps) => {
             />
           </Flex>
         </Flex>
-      </Box>
+      </Box> */}
 
-      <Card $background={"lemon30"} $width={"100%"}>
+      <Card $background={"lemon30"} $width={"100%"} $mt={80}>
```

Comment:
Could replace this with a margin-bottom on the preceiding element

Reply:
Swapped to Mb for proceeding element ✅